css/components/alert.css

```diff
@@ -0,0 +1,66 @@
+@import url(../base/reset.css);
+@import url(../utilities/color.css);
+
+.alert{
+    border: 1px solid;
+    border-radius: 4px;
+    font-size: 1.25rem;
+    padding: 0.5rem;
+    overflow-wrap: anywhere; /* for responsiveness */
```

Comment:
overflow-wrap: break-word will be better?
Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
Done!